content/en/docs/components/katib/user-guides/trial-template.md

```diff
@@ -20,7 +20,7 @@ In Katib examples, you can find the following examples for Trial's Workers:
 
 - [Kubeflow `PyTorchJob`](/docs/components/training/user-guides/pytorch/)
 
-- [Kubeflow `MXJob`](/docs/components/training/user-guides/mxnet)
+- [Kubeflow `PaddleJob`](/docs/components/training/user-guides/paddle)
```

Comment:
We don't have Paddle example in Katib for now: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/kubeflow-training-operator.
Maybe you can just remove the MXJob from this list.